.github/workflows/copybara.yaml

```diff
@@ -0,0 +1,22 @@
+# This script create a Pull request when there is push to copybara branch
+
+name: create-pr-for-copybara
+on:
+  push:
+    branches:
+      - copybara
+jobs:
+  create_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout Copybara branch and Create Pull Request
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        run: |
+          git config --global user.email "yoshi-code-bot@google.com"
+          git config --global user.name "Yoshi Code Bot"
+          git checkout copybara
+          gh pr create \
+            --title "[Copybara] Update with new internal changes from Piper"
+            --body "This is a auto-generated pull request. The PR updates the default branch with new changes from the `copybara branch`. Refer to PR commit history for more details."
```

Comment:
nit: `copybara` branch, not `copybara branch`.

`branch` isn't a part of the name.